src/EdFi.AnalyticsMiddleTier.sln

```diff
@@ -0,0 +1,62 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29411.108
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.AnalyticsMiddleTier.Console", "EdFi.AnalyticsMiddleTier.Console\EdFi.AnalyticsMiddleTier.Console.csproj", "{DA0B242C-2FE6-4314-AA3F-F3BB5DC57130}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.AnalyticsMiddleTier.Tests", "EdFi.AnalyticsMiddleTier.Tests\EdFi.AnalyticsMiddleTier.Tests.csproj", "{839BD756-82E6-4177-96A0-63B2BCB80BCA}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.AnalyticsMiddleTier.DataStandard2", "EdFi.AnalyticsMiddleTier.DataStandard2\EdFi.AnalyticsMiddleTier.DataStandard2.csproj", "{6416D32C-6E0F-4D32-A5D5-3E2350E2A3B0}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.AnalyticsMiddleTier.DataStandard31", "EdFi.AnalyticsMiddleTier.DataStandard31\EdFi.AnalyticsMiddleTier.DataStandard31.csproj", "{47EDBC18-DD4F-4C0A-B67F-5C2A04223592}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.AnalyticsMiddleTier.Common", "EdFi.AnalyticsMiddleTier.Common\EdFi.AnalyticsMiddleTier.Common.csproj", "{E1299B65-6E39-4D6A-A502-B12737ED1645}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.AnalyticsMiddleTier.DataStandard32", "EdFi.AnalyticsMiddleTier.DataStandard32\EdFi.AnalyticsMiddleTier.DataStandard32.csproj", "{283AC53E-C4A5-41AB-90CB-7D0EF9EFF5C2}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution", "Solution", "{DE1426B4-7773-4A66-BC6F-E151B9A265E6}"
```

Comment:
In the ODS Platform they've settled on a folder name of `_Project Information`. The underscore causes the folder to always appear at hte top of the list in Solution Explorer. Can you change to that name?

Reply:
Sure! Fixed.